src/events/instances/MembersJoined.ts

```diff
@@ -27,27 +30,28 @@ class MembersJoined implements Event {
 }
 
 function buildMessage(data: MembersJoinedData) {
-  const { groupId, players } = data;
+  const { groupId, members } = data;
 
-  if (players.length === 1) {
-    const player = players[0];
-    const title = `🎉 New group member: ${player.displayName}`;
+  let content = ``;
 
-    return new MessageEmbed()
-      .setColor(config.visuals.blue)
-      .setTitle(title)
-      .setURL(encodeURL(`https://wiseoldman.net/players/${player.displayName}`));
+  for (let i = 0; i < Math.min(members.length, 10); i++) {
+    const member = members[i];
+    const role = GroupRoleProps[member.role].name;
+
+    content += `${getGroupRoleEmoji(role)} ${member.player.displayName}\n`;
   }
 
-  const url = `https://wiseoldman.net/groups/${groupId}/members`;
-  const title = `🎉 ${players.length} new group members!`;
-  const content = players.map(p => `\`${p.displayName}\``).join(', ');
+  // TODO: Link to the actual page for the activities
```

Comment:
activities will be rendered on the "overview" tab, so no new links are needed (this might change once I start implementing that UI, but for now let's go with that)